### PR TITLE
Feature/look add helio dist

### PR DESCRIPTION
**Pull request by talister**
Adds a column to the LOOK target page with heliocentric distance

### CODE REVIEW

**Comment by jchate6** [comment]
I like the comet tag.

File: neoexchange/core/views.py [hunk 1]
```diff
@@ -1954,8 +1954,11 @@ def build_lookproject_list(disp=None):
                 elif len(subtypes) > 1:
                     body_dict['subtypes'] = ", ".join(subtypes)
                 body_dict['cadence_info'] = body.get_cadence_info()
-                # Compute ephemeris and observability window
+                # Compute ephemeris, distance and observability window
                 emp_line = body.compute_position()
+                if not emp_line:
+                    continue
+                dist_line = body.compute_distances()
                 if not emp_line:
```
Comment (jchate6) [inline neoexchange/core/views.py:1962]: This feels redundant.

Comment (talister) [reply neoexchange/core/views.py]: Because it's a copy/paste error...

**Comment by jchate6** [comment]
I'm getting a failed test: `======================================================================
FAIL: test_add_lookproject_target (neox.tests.test_lookproject_page.LOOKProjectPageTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jchatelain/venv/new_neox/lib64/python3.7/site-packages/mock/mock.py", line 1346, in patched
    return func(*newargs, **newkeywargs)
  File "/home/jchatelain/git/neoexchange/neoexchange/neox/tests/test_lookproject_page.py", line 244, in test_add_lookproject_target
    self.check_for_row_in_table('active_targets', testlines[0])
  File "/home/jchatelain/git/neoexchange/neoexchange/neox/tests/base.py", line 324, in check_for_row_in_table
    self.assertIn(row_text, [row.text.replace('\n', ' ') for row in rows])
AssertionError: '191P Comet Jupiter Family 21 24 28.42 -23 49 36.4 18.6 0.78 Nothing scheduled [-----]' not found in ['191P Comet Jupiter Family 21 24 28.42 -23 49 36.4 18.6 0.78 2.4 Nothing scheduled [-----]', 'C/2013 US10 Comet Hyperbolic, Dynamically New 02 27 17.66 +39 14 13.0 22.3 0.18 15.7 Nothing scheduled [-----]', 'C/2017 K2 Comet Long Period, Dynamically New 19 25 54.88 +32 57 50.2 17.9 0.23 12.0 Inactive since 07/30 [-----]']
`

**Comment by talister** [comment]
> ## I'm getting a failed test: `======================================================================
> FAIL: test_add_lookproject_target (neox.tests.test_lookproject_page.LOOKProjectPageTest)
> 
> Traceback (most recent call last):
> File "/home/jchatelain/venv/new_neox/lib64/python3.7/site-packages/mock/mock.py", line 1346, in patched
> return func(*newargs, **newkeywargs)
> File "/home/jchatelain/git/neoexchange/neoexchange/neox/tests/test_lookproject_page.py", line 244, in test_add_lookproject_target
> self.check_for_row_in_table('active_targets', testlines[0])
> File "/home/jchatelain/git/neoexchange/neoexchange/neox/tests/base.py", line 324, in check_for_row_in_table
> self.assertIn(row_text, [row.text.replace('\n', ' ') for row in rows])
> AssertionError: '191P Comet Jupiter Family 21 24 28.42 -23 49 36.4 18.6 0.78 Nothing scheduled [-----]' not found in ['191P Comet Jupiter Family 21 24 28.42 -23 49 36.4 18.6 0.78 2.4 Nothing scheduled [-----]', 'C/2013 US10 Comet Hyperbolic, Dynamically New 02 27 17.66 +39 14 13.0 22.3 0.18 15.7 Nothing scheduled [-----]', 'C/2017 K2 Comet Long Period, Dynamically New 19 25 54.88 +32 57 50.2 17.9 0.23 12.0 Inactive since 07/30 [-----]']
> `

Fair, given I didn't run that one... I'm out of practice at FTs clearly with the UT-only ETC...

**Review by jchate6 — APPROVED** [review]
Looks good!

**Pull request merged**